cli/lsp/text.rs

```diff
@@ -185,6 +189,20 @@ impl LineIndex {
     }
   }
 
+  pub fn line_length_utf16(&self, line: u32) -> Result<TextSize, AnyError> {
```

Comment:
Probably need a test case for this.

Reply:
I end up removing this function and simplified the logic in `Classifications::to_semantic_tokens` because `start_pos.line == end_pos.line` is always true as there are no multiline tokens